src/Algebra/Solver/IdempotentCommutativeMonoid/Normal.agda

```diff
@@ -19,10 +19,11 @@ open import Data.Bool as Bool using (Bool; true; false; if_then_else_; _∨_)
 open import Data.Fin.Base using (Fin; zero; suc)
 open import Data.Nat.Base using (ℕ)
 open import Data.Vec.Base using (Vec; []; _∷_; lookup; replicate; zipWith)
-import Data.Vec.Relation.Binary.Pointwise.Inductive as Pointwise
+import Data.Vec.Relation.Binary.Pointwise.Inductive as Pointwise using
+  (Pointwise; _∷_; []; Pointwise-≡↔≡; decidable)
 open import Relation.Binary.Definitions using (DecidableEquality)
 import Relation.Binary.Reasoning.Setoid as ≈-Reasoning
-import Relation.Nullary.Decidable       as Dec
+import Relation.Nullary.Decidable as Dec using (map)
```

Comment:
So.. this is a *different* change to that for [`src/Algebra/Solver/CommutativeMonoid/Normal.agda`](https://github.com/agda/agda-stdlib/pull/2614/files#diff-ef42362461b2f36bccc1566aece975accd7c71da7f1df62d34c7cb561852e2a3) above; should they be reconciled?